HtmlMesh/src/html-mesh.ts

```diff
@@ -6,6 +6,114 @@ import { PointerEventsCaptureBehavior } from "./pointer-events-capture-behavior"
+        scalingElement.style.transform = `scale(${scale})`
+        scalingElement.style.visibility = 'visible'
+    }
+}
+
+const SizeFitCoverHandler: SizeFitHandler = {
+    handleElement(element: HTMLElement): HTMLElement {
+        const sizingElement = document.createElement('div')
+        sizingElement.style.display = 'flex'
+        sizingElement.style.justifyContent = 'center'
+        sizingElement.style.alignItems = 'center'
+        sizingElement.style.overflow = 'hidden'
+        const scalingElement = document.createElement('div')
+        scalingElement.style.visibility = 'hidden'
+        scalingElement.appendChild(element)
+        sizingElement.appendChild(scalingElement)
+        return sizingElement
+    },
+    handleSizeFit(parent: HTMLElement, width: number, height: number) {
+        const sizingElement = parent.firstElementChild! as HTMLElement
+        const scalingElement = sizingElement.firstElementChild! as HTMLElement
+        sizingElement.style.width = `${width}px`
+        sizingElement.style.height = `${height}px`
+
+        const [childWidth, childHeight] = [scalingElement!.offsetWidth, scalingElement!.offsetHeight]
+        const scale = Math.max(width / childWidth, height / childHeight)
+        scalingElement.style.transform = `scale(${scale})`
+        scalingElement.style.visibility = 'visible'
+    }
+}
+
+
+const SizeFitStretchHandler: SizeFitHandler = {
+    handleElement(element: HTMLElement): HTMLElement {
+        const sizingElement = document.createElement('div')
+        sizingElement.style.display = 'flex'
+        sizingElement.style.justifyContent = 'center'
+        sizingElement.style.alignItems = 'center'
+        const scalingElement = document.createElement('div')
+        scalingElement.style.visibility = 'hidden'
+        scalingElement.appendChild(element)
+        sizingElement.appendChild(scalingElement)
+        return sizingElement
+    },
+    handleSizeFit(parent: HTMLElement, width: number, height: number) {
+        const sizingElement = parent.firstElementChild! as HTMLElement
+        const scalingElement = sizingElement.firstElementChild! as HTMLElement
+        sizingElement.style.width = `${width}px`
+        sizingElement.style.height = `${height}px`
+
+        const [childWidth, childHeight] = [scalingElement!.offsetWidth, scalingElement!.offsetHeight]
+        scalingElement.style.transform = `scale(${width / childWidth}, ${height / childHeight})`
+        scalingElement.style.visibility = 'visible'
+    }
+}
+
+const SizeFitNoneHandler: SizeFitHandler = {
+    handleElement(element: HTMLElement): HTMLElement {
+        return element
+    },
+    handleSizeFit(parent: HTMLElement, width: number, height: number) {
+        const childElement = parent.firstElementChild as HTMLElement;
+        if (childElement) {
+            childElement.style.width = `${width}px`;
+            childElement.style.height = `${height}px`;
+        }
+    }
+}
+
+const SizeFitHandlerMap = {
```

Comment:
Can we move all of the FitStrategy stuff into its own ts file.  Then we would export the FitStrategy type as well as FitStrategy.CONTAIN, FitStrategy.COVER, etc...  (Similar to how the RenderingGroup compare functions are exposed).